templates/certified/model-details.html

```diff
@@ -118,7 +118,7 @@ <h4>Hardware</h4>
                   <tr>
                     <th colspan="2" class="u-text--muted">{{ hardware_subtitle }}</th>
                     <td colspan="8">{% for value in values %} 
-                      <p style="margin-bottom: 0.5rem;">{{ value.name }} {{ value.bus }}({{ value.identifier }})</p>
+                      <p style="margin-bottom: 0.5rem;">{{ value.name }} {{ value.bus }}{% if value.bus in ["usb", "pci"] %} ({{ value.identifier }}){% endif %}</p>
```

Comment:
I think I'd put the bus inside the conditional as well. I don't think it is valuable to show the bus type when there is no identifier.

Reply:
Sure. Done.